validator/tests/test_static.py

```diff
@@ -197,24 +165,23 @@ def test_non_consecutive_duplicates(self):
         df = pd.DataFrame([["a", "1"], ["b", "2"], ["a", "1"]])
         validator.check_duplicate_rows(df, "file", report)
         assert len(report.raised_warnings) == 1
-        assert report.raised_warnings[0].expression == [2]
-        assert report.raised_warnings[0].check_data_id[1] == "file"
+        assert report.raised_warnings[0].check_name == "check_duplicate_rows"
 
     def test_multiple_distinct_duplicates(self):
         validator = StaticValidator(self.params)
         report = ValidationReport([])
```

Comment:
Update all references to ValidationReport to use sets.

Reply:
Done